.github/workflows/mavenCentral.yml

```diff
@@ -7,29 +7,33 @@ on:
         required: true
 jobs:
   build:
-    name: Maven Central Sync
+    name: "Maven Central Sync"
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+      - name: "📥 Checkout repository"
+        uses: actions/checkout@v4
         with:
           ref: v${{ github.event.inputs.release_version }}
-      - uses: gradle/wrapper-validation-action@v2
-      - name: Set up JDK
-        uses: actions/setup-java@v4.2.1
+      - name: "☕️ Setup JDK"
+        uses: actions/setup-java@v4
```

Comment:
Not sure about using `@v4` instead of the exact version with `@v4.2.1`. I had seen issues with GitHub actions updating and breaking builds.